<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IFace Put your face in machice</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/app_icon.jpg</normaloff>icon/app_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 49, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>870</width>
      <height>621</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QLabel" name="video_feed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-40</y>
       <width>870</width>
       <height>661</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>621</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>ACTIONS</string>
     </property>
     <widget class="QPushButton" name="register_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recognize_face_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Recognize</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
  <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>719</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
